Packages/com.unity.inputsystem/InputSystem/Plugins/UI/InputSystemUIInputModule.cs

```diff
@@ -387,7 +388,10 @@ private void ProcessPointerMovement(ExtendedPointerEventData eventData, GameObje
             if (currentPointerTarget == null || eventData.pointerEnter == null)
             {
                 for (var i = 0; i < eventData.hovered.Count; ++i)
+                {
+                    onPointerExit?.Invoke(eventData.hovered[i], eventData);
```

Comment:
For all these events, would probably make sense to allow the handler to call `Use` to "consume" the event and prevent `ExecuteEvents` from being run.

Reply:
You mean optionally, correct? I don't think all use-cases will want to fully 'consume' the events, plus there is no mechanism currently in place to prevent subsequent callbacks from being invoked if the prior callback 'consumed' the event.